.github/workflows/release.yaml

```
@@ -37,16 +37,41 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: "3.11"
+      - name: Install Pipenv
+        run: |
+          python -m pip install pipenv
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip wheel
-          python -m pip install '.[dev]'
-      - name: Write version and commit hash
-        run: write_release_info -p src/usdb_syncer/constants.py -v ${{ github.ref_name }} -c ${{ github.workflow_sha }}
+          pipenv sync --system
+      - name: Prepend src to PYTHONPATH (Windows)
+        run: |
+          echo "PERSISTENT_PYTHONPATH=$(pwd)\src;$env:PYTHONPATH" >> $GITHUB_ENV
+        if: matrix.TARGET == 'Windows'
+      - name: Prepend src to PYTHONPATH (Linux/macOS)
+        run: |
+          echo "PERSISTENT_PYTHONPATH=$(pwd)/src:$PYTHONPATH" >> $GITHUB_ENV
+        if: matrix.TARGET == 'Linux' || matrix.TARGET == 'macOS'
+      - name: Write version and commit hash (Windows DEBUG)
+        run: |
+          echo $env:PYTHONPATH
+          echo $PYTHONPATH
+          python -m src.tools.write_release_info -p src/usdb_syncer/constants.py -v ${{ github.ref_name }} -c ${{ github.workflow_sha }}
```

Did you try without the `-m` flag?
```suggestion
        run: python ./src/tools/write_release_info.py -p src/usdb_syncer/constants.py -v ${{ github.ref_name }} -c ${{ github.workflow_sha }}
```
It should be possible without modifyng the PYTHONPATH.

---

I did and had the same problem of Module 'tools' cannot be found